source/2019-06-01-the-ember-times-issue-101.md

```diff
@@ -11,15 +11,37 @@ responsive: true
 <SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
 
 <SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
+This week:
+Put your thoughts to paper for the #EmberJS2019 Roadmap RFC 🛣🗺
+...
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
-<change section title emoji>
-<consider adding some bold to your paragraph>
+## [#EmberJS2019 🛣🗺](https://blog.emberjs.com/2019/05/20/ember-2019-roadmap-call-for-posts.html)
   
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+🚨 Blog posts wanted! 🚨 
+
+We would like you to write a blog post to propose **goals and direction** for Ember for the next year. The content of these posts will help the Core Team to draft their **next Roadmap RFC**. Please share your posts by **Monday, July 17th**. 
```

Comment:
Can we also mention GitHub gists and tweets as alternatives to blog posts? I think the deadline is June 17th, to be considered as part of RFC.

Reply:
fixed both - geez, that would have been a bad error saying July! 😅